src/UI/Pages/AgentProcess/Page.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import { StoreProvider } from "easy-peasy";
+import { Either } from "@/Core";
+import {
+  AgentProcessStateHelper,
+  GetAgentProcessQueryManager,
+  getStoreInstance,
+  QueryResolverImpl,
+} from "@/Data";
+import {
+  AgentProcessData,
+  DeferredApiHelper,
+  dependencies,
+  DynamicQueryManagerResolver,
+  StaticScheduler,
+} from "@/Test";
+import { DependencyProvider } from "@/UI/Dependency";
+import { UrlManagerImpl } from "@/UI/Utils";
+import { Page } from "./Page";
+
+function setup() {
+  const store = getStoreInstance();
+  const scheduler = new StaticScheduler();
+  const apiHelper = new DeferredApiHelper();
+  const queryResolver = new QueryResolverImpl(
+    new DynamicQueryManagerResolver([
+      new GetAgentProcessQueryManager(
+        apiHelper,
+        new AgentProcessStateHelper(store)
+      ),
+    ])
+  );
+  const urlManager = new UrlManagerImpl(
```

Comment:
`dependencies` already provides an urlManager.
Do you need this to explicitly set the environment to `environment` instead of `env`.